python_practice/python_practice.md

```diff
@@ -0,0 +1,147 @@
+estimated_time: 1 hour
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Import a dataset from an online database
+- Recode data and change variable types in a dataframe
+- Use exploratory data visualization to identify trends in data and generate hypotheses
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Python Practice
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?:** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** Learners should be familiar with [the basics of Python coding](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/python_basics_writing_python_code/python_basics_writing_python_code.md), including [data transformation with pandas](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/pandas_transform/pandas_transform.md) and [data visualization with matplotlib and seaborn](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/data_visualization_in_seaborn/data_visualization_in_seaborn.md). Learners should also have access to Python, either on their own computer or in the cloud. 
+
+**Learning Objectives**:
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+For this module, you'll need access to Python, either on your own computer or in the cloud. For details about how to download Python or use Google Colab (a cloud-based notebook environment), you can take a look at the ["Accessing Python" section of our Demystifying Python module](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_python/demystifying_python.md#9). 
+
+## Topics to Review
+
+- Importing Python libraries 
+- Reading a CSV and creating a data frame
+- Editing/recoding data
+- Exploratory data visualization
+
+Stretch topics
+=====
+
+There are a couple of additional skills that haven't been introduced in a module yet, but that might come in handy solving this data science problem: 
+
+- Correlation matrices
+- Heat maps
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Finding correlations and creating heat maps are not topics that we have taught you in the preceding modules-- which means you might need to do some searching to figure out how to do them! This is an important step in your development as a programmer and a data scientist, but don't worry if it seems difficult in the beginning; it'll get easier with practice. To start, try googling "Python" and "correlations" and see where it takes you. 
```

Comment:
```suggestion
Finding correlations and creating heat maps are not topics that you have learned in preceding modules or in other prior learning -- which means you might need to do some searching to figure out how to do them! This is an important step in your development as a programmer and a data scientist, but don't worry if it seems difficult in the beginning; it'll get easier with practice. To start, try googling "Python" and "correlations" and see where it takes you. 
```
de-coupling this a little from our other modules in case this is ever delivered separately